### DISCUSSION TITLE
Not updating properly

### DISCUSSION THREAD

**Discussion by Shuyallis** [lnreader/lnreader#1063] [Q&A] [answered]
I have just updated the app an hour ago but I seem to be facing some issues while updating stories from lightnovelpub as we have know it's categorized per 100 chap but the story has already reached 202 but the chaps in the pauses at 200 please help since my other added story also won't update properly I think this problem is for the lightnovelpub site only since I haven't encountered some problem while using the other extension

**Comment by nyagami** [comment, marked as answer]
It is known issues and has been fixed, app v2.0.0

**Reply by Shuyallis** [reply]
Really thanks I'll check the app

**Reply by nyagami** [reply]
the fix will come in next release btw

**Reply by nyagami** [reply]
Or you can try this dev build https://github.com/LNReader/lnreader/actions/runs/8896804452

**Reply by Shuyallis** [reply]
Owww, I thought my phone was the problem anyways thanks for clarifying
